common/djangoapps/course_modes/models.py

```diff
@@ -26,6 +26,25 @@
                   ])
 
 
+class EnrollmentMode(models.Model):
```

Comment:
@bderusha what do you think of this model (and admin stuff above)? Do you like the name? Can you think of any way to link mode_slug in this model to mode_slug in CourseMode below?